README.md

```diff
@@ -168,6 +168,3 @@ Much of the SDK code is generated from metadata specs about the REST APIs. Do no
   - File an issue describing the problem,
   - Refer to the the [AutoRest project](https://github.com/azure/autorest) to view and modify the generator, or
   - Add additional methods, properties, and overloads to the SDK by adding classes in the 'Customizations' folder of a project
-
```

Comment:
This was added by a recent merge please ensure you merge correctly with it.

Reply:
Rebase to upstream this morning seems to have cleared this.   My goal is to have the opportunity to do a final rebase before the PR gets merged to keep any of this kind of drift to a minimum.